Makefile

```diff
@@ -16,7 +16,7 @@ OPENSHIFT=${CURDIR}/../../github.com/openshift/release
 
 install:
 	for img in $(CORE_IMAGES); do \
-		go install -tags="disable_gcp,disable_aws,disable_azure" $$img ; \
+		go install $$img ; \
```

Comment:
`tags="disable_gcp,disable_aws,disable_azure` is this covered elsewhere?

Reply:
No. TBH. I don't know why they were there. Maybe for historical reasons? Searching for these tags through the codebase, I did not find anything. Do you have an idea?

Reply:
This was meant for downstream only we were saying failures without this in the past. Here is a related file https://github.com/google/go-containerregistry/blob/13e1a6b06afd59fbc54e94ddcfb1475504e72883/pkg/authn/k8schain/k8schain_aws.go but let me check if this was existed only on past versions.

Reply:
Interesting. 
This was reimplemented in https://github.com/google/go-containerregistry/pull/1234 on Jan 8 2022 and should be included in version v0.9.0.
Our repo uses go-containerregistry v0.13.0.
I suppose we should be fine but not 100% sure

Reply:
I only see the related files in very old versions eg. [here](https://github.com/google/go-containerregistry/blob/v0.8.0/pkg/authn/k8schain/k8schain_aws.go). I found this old issue: https://github.com/knative/serving/issues/10878#issue-821067931. Still digging.

Reply:
However given what I read here: https://github.com/knative/serving/tree/release-1.15/vendor/github.com/google/go-containerregistry/pkg/authn/k8schain.

> In addition to those, the keychain also includes cloud-specific credential helpers for Google Container Registry (and Artifact Registry), Azure Container Registry, and Amazon AWS Elasic Container Registry. This means that if the keychain is used from within Kubernetes services on those clouds (GKE, AKS, EKS), any available service credentials will be discovered and used.
In general this keychain should be used when the code is expected to run in a Kubernetes cluster, and especially when it will run in one of those clouds. To get a cloud-agnostic keychain, use [pkg/authn/kubernetes](https://github.com/knative/serving/blob/release-1.15/vendor/github.com/google/go-containerregistry/pkg/authn/kubernetes) instead.

I am wondering if now this  cloud specific overhead cannot be disabled or was removed.

Reply:
Searching the go-containerregistry repo at version v0.15.0 for disable_azure, disable_gcp, disable_aws. It doesn't find anything. So, I doubt the tags have any effect now.

Reply:
Yeah also check my comment here in our [discussion](https://redhat-internal.slack.com/archives/CF5ANN61F/p1725609088652819?thread_ts=1725364430.634709&cid=CF5ANN61F) about the overhead though.